site/routes.js

```diff
@@ -23,39 +31,74 @@ import { orderSections } from './order-sections';
 const routes = [
   {
     name: 'Home',
-    route: '/'
+    route: '/',
+    component: Home
   },
   {
     name: 'Documentation',
-    route: '/documentation/'
+    route: '/documentation/',
+    component: Documentation
+  },
+  {
+    name: 'Examples',
+    route: '/examples/',
+    component: Examples,
+    children: [{
+      name: 'Inputs',
+      route: '/examples/inputs/',
+      component: ExampleInputs
+    }, {
+      name: 'Map components',
+      route: '/examples/map-components/',
+      component: ExampleMapComponents
+    }, {
+      name: 'Miscellaneous',
```

Comment:
Not sure about the miscellaneous category. There is good stuff in here, I think we should have _more_ categories – will be easier for ppl to find what they are looking for: ('tooltip', 'badges', 'progress bar' as separate pages).